test/proxy/transparent/ProxyAdmin.test.js

```diff
@@ -51,18 +40,8 @@ contract('ProxyAdmin', function (accounts) {
 
     it('changes proxy admin', async function () {
       await this.proxyAdmin.changeProxyAdmin(this.proxy.address, newAdmin, { from: proxyAdminOwner });
-      expect(await this.proxy.admin.call({ from: newAdmin })).to.eq(newAdmin);
-    });
-  });
-
-  describe('#getProxyImplementation', function () {
-    it('returns proxy implementation address', async function () {
-      const implementationAddress = await this.proxyAdmin.getProxyImplementation(this.proxy.address);
-      expect(implementationAddress).to.be.equal(this.implementationV1.address);
-    });
-
-    it('call to invalid proxy', async function () {
-      await expectRevert.unspecified(this.proxyAdmin.getProxyImplementation(this.implementationV1.address));
+      // Allows new admin to call ifAdmin functions
+      await this.proxy.changeAdmin(newAdmin, { from: newAdmin });// expect to not revert
```

Comment:
Instead of this, to verify that the admin changed we should read the EIP-1967 storage slot.